lib/linguist/heuristics.rb

```diff
@@ -136,6 +136,14 @@ def call(data)
       end
     end
 
+    disambiguate ".eslintrc" do |data|
+      if /^{/.match(data)
```

Comment:
Could you add a test in [`test/test_heuristics.rb`](https://github.com/github/linguist/blob/master/test/test_heuristics.rb) for that heuristic rule?